client/src/App.js

```diff
@@ -213,9 +213,10 @@ class App extends Component {
     }
   }
 
-  async refreshProfile(role, id) {
```

Comment:
If you merge this first, I will have to modify calls to refreshProfile in AlumniProfile. This change makes sense though!

Reply:
I can actually change this back - I was attempting to use refreshProfile with NavBar but ended up not using it.